framework/source/class/qx/event/handler/Mouse.js

```diff
@@ -375,6 +377,17 @@ qx.Class.define("qx.event.handler.Mouse",
 
       this.__fireEvent(domEvent, type, target);
 
+      /*
+       * In order to normalize middle button click events we
+       * need to fire an artificial click event if the client
+       * fires auxclick events for non primary buttons instead.
+       * 
+       * See https://github.com/qooxdoo/qooxdoo/issues/9268
+       */
+      if (type == "auxclick" && domEvent.button == 1) {
+        this.__fireEvent(domEvent, "click", target);
```

Comment:
The api documentation says that a click event will be fired on a left and middle mouse button click. See http://www.qooxdoo.org/devel/api/#qx.ui.core.Widget~click!event
The PR does exactly that, it fires an artificial middle-mouse-button-click if a middle-mouse-button-auxclick ist detected.